resample-1.8.1/src/endian.h

```diff
@@ -4,6 +4,8 @@
 #include <sys/_endian.h>
 #elif defined __linux__
 #include <endian.h>
+#elif defined __HAIKU__
```

Comment:
What exactly is haiku? Isn't it somehow linux-ish and would/should define `__linux__`? Anyway plese merge with above: `|| defined __HAIKU__`

Reply:
Nope, haiku doesnt masks itself to other os, it defines only __HAIKU__